src/main/java/edu/byu/cs/util/RepoUrlValidator.java

```diff
@@ -74,7 +83,7 @@ public String cleanRepoUrl(@Nullable String repoUrl) throws InvalidRepoUrlExcept
      * </ol>
      * @return An array of strings representing regexes.
      */
-    private String[] getRepoRegexPatterns() {
+    private static String[] getRepoRegexPatterns() {
```

Comment:
Ditto. Static management of data structures makes the class less adaptable to change and customization. This is a design regression that should not be accepted.